PennMobile/Dining/Model/PastDiningBalances.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  PastDiningBalances.swift
+//  PennMobile
+//
+//  Created by Andrew Antenberg on 3/25/22.
+//  Copyright © 2022 PennLabs. All rights reserved.
+//
+
+import Foundation
+
+struct PastDiningBalances: Codable {
+    let balanceList: [DiningBalance]
+}
```

Comment:
I don't think it is necessary to create a new struct just to be a wrapper for an array of another struct.

Reply:
When decoding, you can use `JSONDecoder().decode([DiningBalance].self, from: {})`

Reply:
Resolved: it is required due to the API response type